Formula/lima.rb

```diff
@@ -4,6 +4,8 @@ class Lima < Formula
   url "https://github.com/lima-vm/lima/archive/v0.8.3.tar.gz"
   sha256 "8c23daa9ff3835bc47b187c47202ff95efd6a767f58789c11b2674b4ae0da403"
   license "Apache-2.0"
+  revision 1
```

Comment:
```suggestion
```
We don't need to make users reinstall `lima` when they do `brew upgrade` for this change, so let's omit the `revision`.

Reply:
omitted